src/SwedbankPay.Sdk/Common/TypeSafeEnum.cs

```diff
@@ -20,14 +20,11 @@ public abstract class TypeSafeEnum<TEnum, TValue> : IEquatable<TypeSafeEnum<TEnu
                 // multiple enums with same value are allowed but store only one per value
                 var dictionary = new Dictionary<TValue, TEnum>();
                 foreach (var item in GetAllOptions())
-                    if (!dictionary.ContainsKey(item.value))
-                        dictionary.Add(item.value, item);
+                    if (!dictionary.ContainsKey(item.Value))
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)